src/main/java/edu/kis/powp/jobs2d/command/CommandVisitor.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.command;
+
+public class CommandVisitor implements Visitor {
+	private int counterICompoundCommand = 0;
+	private int counterOperateToCommand = 0;
+	private int counterSetPositionCommand = 0;
+
+
+	public void visitICompoundCommand(ICompoundCommand iCompoundCommand) {
+		this.counterICompoundCommand++;
```

Comment:
this visitor should iterate over subcommands of iCompoundCommand and execute accept.